src/curl.c

```diff
@@ -478,7 +478,7 @@ static int swupd_curl_get_file_full(const char *url, char *filename,
 		local.path = filename;
 
 		if (resume_ok && resume_download_supported && lstat(filename, &stat) == 0) {
-			fprintf(stderr, "Curl: Resuming download for '%s'\n", url);
+			print("Curl: resuming download for '%s'\n", url);
```

Comment:
same here

Reply:
For debug and error messages I agree with you, but not for this one. Print doesn't add any prefix. So error/debug messages are going to be printed like:

    Error: curl - some error message
    2019-01-18 18:34:52 (src/curl.c:513)              Debug: curl - debug message

and print are like:

    Curl: this is a message

I think that this would be weird:

    curl - this is a message

What do you think?

Reply:
ooh ok, it makes sense, carry on then... :)